test/unit/c_api/eigensolver/test_eigensolver_c_api_wrapper.c

```diff
@@ -0,0 +1,57 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include "test_eigensolver_c_api_wrapper.h"
+
+#include <dlaf_c/desc.h>
+#include <dlaf_c/eigensolver/eigensolver.h>
+#include <dlaf_c/utils.h>
+
+#ifdef DLAF_WITH_SCALAPACK
+void C_dlaf_pdsyevd(char uplo, int m, double* a, int ia, int ja, int* desca, double* w, double* z,
+                    int iz, int jz, int* descz, int* info) {
+  dlaf_pdsyevd(uplo, m, a, ia, ja, desca, w, z, iz, jz, descz, info);
+}
+
+void C_dlaf_pssyevd(char uplo, int m, float* a, int ia, int ja, int* desca, float* w, float* z, int iz,
+                    int jz, int* descz, int* info) {
+  dlaf_pssyevd(uplo, m, a, ia, ja, desca, w, z, iz, jz, descz, info);
+}
+
+void C_dlaf_pzheevd(char uplo, int m, dlaf_complex_z* a, int ia, int ja, int* desca, double* w,
+                    dlaf_complex_z* z, int iz, int jz, int* descz, int* info) {
+  dlaf_pzheevd(uplo, m, a, ia, ja, desca, w, z, iz, jz, descz, info);
+}
+
+void C_dlaf_pcheevd(char uplo, int m, dlaf_complex_c* a, int ia, int ja, int* desca, float* w,
+                    dlaf_complex_c* z, int iz, int jz, int* descz, int* info) {
+  dlaf_pcheevd(uplo, m, a, ia, ja, desca, w, z, iz, jz, descz, info);
+}
```

Comment:
What is the reason for this wrappers?

Reply:
See discussion in https://github.com/eth-cscs/DLA-Future/pull/886#discussion_r1211306641. Do I need to add a comment?

Reply:
Yes please.